dags/bqetl_artifact_deployment.py

```diff
@@ -119,7 +138,8 @@ def check_for_queued_runs(dag_id: str) -> bool:
         task_id="publish_views",
         cmds=["bash", "-x", "-c"],
         arguments=[
-            "script/bqetl view publish --add-managed-label --skip-authorized --project-id=moz-fx-data-shared-prod && "
+            generate_sql_cmd_template
```

Comment:
Could we avoid running SQL generation three separate times?  Maybe we could have a conditional task that does the SQL generation and pushes it to the `private-generated-sql` branch if the param is set, then the publish tasks could conditionally pull down the latest files from that branch if the param is set?